### ISSUE TITLE
New release ?

### ISSUE THREAD

**Issue by frallain** [jazzband/django-push-notifications#636]
Would it be possible to enjoy a new release of django-push-notifications ? It has been one year and half since no release and I can see several commits the community could benefit from, given that Django 4.0 is out and that the current 2.0.0 release emits the following  warnings :
```
/venv/lib/python3.8/site-packages/push_notifications/models.py:235: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    verbose_name=_("User public encryption key"),
```
Thanks for this package, keep up the good work!

**Comment by bertonha** [comment]
Would be really nice to have a new release because the current released version does not supports Django 4.0.
it raise the following error.

```
File "push_notifications/admin.py", line 3, in <module>
   from django.utils.encoding import force_text
   
ImportError: cannot import name 'force_text' from 'django.utils.encoding'
```

**Comment by richardthegit** [comment]
Just to add - it also doesn't work with current versions of PyJWT, which make life very difficult when using other packages rely on the new PyJWT API.

**Comment by bradleydworak** [comment]
Ditto, code needs updated for compatibility with Django 4.0.
The following conflicting lines:
```python
from django.utils.encoding import force_text
from django.utils.translation import ugettext_lazy as _
```
should be updated to:
```python
from django.utils.encoding import force_str
from django.utils.translation import gettext_lazy as _
```

**Comment by bigfootjon** [comment]
The required fixes seem to already be present: https://github.com/jazzband/django-push-notifications/commit/ff1337798ce00a059edeccb9702c1c6db61a6bf8

we just need a new release created

**Comment by sevdog** [comment]
The `force_text` was replaced with `force_str` in ce2140fccecb83cb2bf4203424bbdd47f44428ea.

**Comment by bradleydworak** [comment]
Thank you @bigfootjon and @sevdog, I concur. We are awaiting a release. Any known issues holding it up?

**Comment by gabn88** [comment]
@simonkern Would it be possible to create a new release?

**Comment by simonkern** [comment]
I only opened two or three pull requests, but have no permissions regarding this package.

**Comment by gabn88** [comment]
@simonkern  Thanks for the fast response. Do you know who has? It is a really nice package, and it would be a pity to lose it due to the lack of new releases.

**Comment by jamaalscarlett** [comment]
Sorry everyone, I have pypi access

**Comment by jamaalscarlett** [comment]
Which commit are we using for the release?

**Comment by bigfootjon** [comment]
Latest commit from ‘master’ seems fine to me. It has some missing migrations in it. 4.0 support landed a few commits before that

**Comment by jamaalscarlett** [comment]
OK I'll use db0606c908b669d9803cdfc26fca0926cb059fd3

**Comment by jamaalscarlett** [comment]
It will be 2.1.0

**Comment by daveisfera** [comment]
Thanks for getting this out, but if it's not too late, the version number should probably be 3.0.0 since it's dropping support for Django 1.x:
https://github.com/jazzband/django-push-notifications/compare/2.0.0...master#diff-fa602a8a75dc9dcc92261bac5f533c2a85e34fcceaff63b3a3a81d9acde2fc52R32

**Comment by vvxhid** [comment]
any updates?

**Comment by jamaalscarlett** [comment]
@jleclanche could you create this release.  Apparently when I restored my phone, it reset my authenticator app so I am currently locked out of pypi

**Comment by bradleydworak** [comment]
@jleclanche and @jamaalscarlett Is there something in particular about not feeling comfortable with a new release? Not clear what the hold up is.

**Comment by jleclanche** [comment]
@bradleydworak I don't feel comfortable doing a release on this project because I haven't done one in years and I don't use the library anymore to test it / know if it's ready for a release.

**Comment by bertonha** [comment]
@jleclanche I am using the latest master for a while, and its sending push-notifications to IOS and Android.

The current version 2.0.0 does not support Django 4.0 which is a blocker for many people. and a new release would unblock it.

**Comment by jleclanche** [comment]
Again: If another jazzband member wants to step up to do the release, I can give permissions. @jamaalscarlett I can also add whatever account of yours you need. But I am not comfortable doing it myself.

**Comment by daveisfera** [comment]
Could the version be bumped to 3.0 and it released? That would be lower risk since people shouldn't be hit with the package upgrading under them and then more testing can happen for those that are ready to make the upgrade.

**Comment by jamaalscarlett** [comment]
Sorry about the delay everyone
https://github.com/jazzband/django-push-notifications/releases/tag/3.0.0
https://pypi.org/manage/project/django-push-notifications/release/3.0.0/
